src/engine/server/server.cpp

```diff
@@ -1444,7 +1444,7 @@ int CServer::Run()
 			PumpNetwork();
 
 			// wait for incoming data
-			m_NetServer.Wait(clamp(int((TickStartTime(m_CurrentGameTick+1)-time_get())*1000/time_freq()), 1, 1000/SERVER_TICK_SPEED/2));
+			m_NetServer.Wait(clamp(int((TickStartTime(m_CurrentGameTick+1)-Now)*1000/time_freq()), 1, 1000/SERVER_TICK_SPEED/2));
```

Comment:
This one seems to be timing sensitive; if the game loop took a nonzero amount of time, we'll be waiting exactly this amount of time too long.

Reply:
reverted this one :)